core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestionBuilder.java

```diff
@@ -362,4 +366,30 @@ protected XContentBuilder innerToXContent(XContentBuilder builder, Params params
         }
         return builder;
     }
+
+    @Override
+    public String getWriteableName() {
+        return SUGGESTION_NAME;
+    }
+
+    @Override
+    public void doWriteTo(StreamOutput out) throws IOException {
+        // TODO
```

Comment:
@MaineC Exception is maybe a good idea. This however is PR against the feature branch, so this doesn't go into master and in order to share some framework code quickly we said we could merge this maybe with some open issues for now.